### ISSUE TITLE
Make Projects a Seperate Page

### ISSUE THREAD

**Issue by 0xadityaa** [0xadityaa/Portfolio#9]
Detach the Projects page from the about "/" route as this violates accessibility best-practice as well as damages the SEO, UX.

<img width="388" alt="Image" src="https://github.com/user-attachments/assets/737a42a8-950e-4181-9c4e-4d087ccc87ed" />
